src/main.c

```diff
@@ -177,7 +203,9 @@ Please check your configuration file and permissions\
       return result;
     }
   }
-  else if (! cli_user_options.want_purge && ! cli_user_options.want_empty_trash && ! init_data_dir)
+  else if (! cli_user_options.want_purge &&
```

Comment:
My gut feeling about the logic here is that it should just be an `else`. I think this because this last print is the "catch all" if none of the other procedures are executed. However, it's fine as it is.

Reply:
That sounds right, and a little better coding even though it won't change the result. Nice catch.

Reply:
I tried that change but the check failed. I'll investigate further another time ;)